src/components/Todo/Todo.js

```diff
@@ -14,13 +14,13 @@ const Todo = ({
   status,
   onStatusChange,
   onDelete,
-  onUpdateText,
+  onUpdateTitle,
```

Comment:
Para seguir la convención, el evento debería ir después del sujeto del mismo

```suggestion
  onTitleUpdate,
```